examples/SharedMemory/plugins/eglPlugin/eglRendererVisualShapeConverter.cpp

```diff
@@ -1565,6 +1572,24 @@ void EGLRendererVisualShapeConverter::removeVisualShape(int collisionObjectUniqu
 		delete ptr;
 		m_data->m_swRenderInstances.remove(collisionObjectUniqueId);
 	}
+
```

Comment:
In the case that we give `removeVisualShape` an `collisionObjectUniqueId` we would then need to iterate over the links and remove them. This would be instead of doing it in `PhysicsServerCommandProcessor.cpp:L11226`